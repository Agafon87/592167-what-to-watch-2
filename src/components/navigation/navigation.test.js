import React from "react";
import renderer from "react-test-renderer";
import NavigationMoviePageTabs from "./navigation.jsx";

const mock = {
  tabItemList: [],
  handleMoviePageTabClick: jest.fn(),
  filmTab: ``,
};

describe(`Navigation component`, () => {
  const {tabItemList, handleMoviePageTabClick, filmTab} = mock;

  it(`renders correctly`, () => {
    const tree = renderer.create(<NavigationMoviePageTabs
      tabItemList={tabItemList}
      handleMoviePageTabClick={handleMoviePageTabClick}
      filmTab={filmTab}
    />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

