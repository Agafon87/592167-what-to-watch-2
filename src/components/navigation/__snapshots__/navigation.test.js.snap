// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationMoviePageTabs correctly renderer after test 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  />
</nav>
`;
